boards/sodaq-explorer/include/periph_conf.h

```diff
@@ -190,27 +105,6 @@ static const adc_conf_chan_t adc_channels[] = {
 #define ADC_NUMOF                           ARRAY_SIZE(adc_channels)
 /** @} */
 
-/**
- * @name SPI configuration
- * @{
- */
-static const spi_conf_t spi_config[] = {
-    {
-        .dev      = &SERCOM3->SPI,
-        .miso_pin = GPIO_PIN(PA, 22),
-        .mosi_pin = GPIO_PIN(PA, 20),
-        .clk_pin  = GPIO_PIN(PA, 21),
-        .miso_mux = GPIO_MUX_C,
-        .mosi_mux = GPIO_MUX_C,
```

Comment:
I think there was an existing mistake on pin MUX for MOSI and CLK.

Reply:
Indeed